engine/src/flutter/shell/platform/embedder/embedder_semantics_update.cc

```diff
@@ -279,6 +320,7 @@ void EmbedderSemanticsUpdate2::AddNode(const SemanticsNode& node) {
       increased_value_attributes.attributes,
       decreased_value_attributes.count,
       decreased_value_attributes.attributes,
+      &(flags_.back()),
```

Comment:
No need for the extra brackets, just:
```suggestion
      &flags_.back(),
```